### PR TITLE
{devel}[fosscuda/2020b] PyTorch v1.9.0 w/ Python 3.8.6 and IMKL

### PR DESCRIPTION
**Pull request by terjekv**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by terjekv** [comment]
Depends on https://github.com/easybuilders/easybuild-easyconfigs/pull/13697

**Comment by terjekv** [comment]
The three tests that are added to the exception list, `distributed/test_distributed_fork`,  `distributed/test_distributed_spawn`, and `test_optim` fail on A10 cards with Cuda Compute Capability 8.6. This failure does not seem to be IMKL-related as @zilverberg found similar issues with the default 1.9.0-build on A100 cards.

**Comment by Flamefire** [comment]
Requires https://github.com/easybuilders/easybuild-easyconfigs/pull/13697

Please add short description why the tests fail as comments to the EC and a link to a TF issue where applicable

**Comment by terjekv** [comment]
> Requires #13697
> 
> Please add short description why the tests fail as comments to the EC and a link to a TF issue where applicable

Will do. I need to get a few other things done today, but I'll go through my notes and logs and document the failures.

File: easybuild/easyconfigs/p/PyTorch/PyTorch-1.9.0-fosscuda-2020b-imkl.eb [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    # PyTorch-1.7.1_correctly-pass-jit_opt_level.patch
+    'd4d967d47f8a6172fcbf57f0a61835482968850967c4fdb01108b720696a988d',
+    # PyTorch-1.7.1_fix-alias-violation-in-bitwise-ops.patch
+    'e92f054f1297df83ace901e7af38222787b709ee29580f5f2b89a300ca03666b',
+    '89ac7a8e9e7df2e64cf8404fe3a279f5e9b759fee41c9de3aaff9c22f385c2c6',  # PyTorch-1.8.1_dont-use-gpu-ccc-in-test.patch
+    'eca718ce0ecc61b57659808179cc660919b3c25b5dce326f51c1cc5a5bfb9829',  # PyTorch-1.8.1_fix-arange-on-VSX.patch
+    # PyTorch-1.8.1_fix-faulty-asserts-and-skip-test.patch
+    '1601eacd336e01176bdbdbc5a2207743dc2999b535a738a8e36b3da3e8a2f3b6',
+    # PyTorch-1.8.1_increase-distributed-test-timeout.patch
+    'c94eda2289692d00873e2ae8cecaaf7a1e5e657dfa4cfcd94b56093f6f4766ad',
```
Comment (Flamefire) [inline easybuild/easyconfigs/p/PyTorch/PyTorch-1.9.0-fosscuda-2020b-imkl.eb:57]: Will conflict with https://github.com/easybuilders/easybuild-easyconfigs/pull/13700
So whichever gets merged first requires a change to the other

**Review by Flamefire — CHANGES_REQUESTED** [review]

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
taurusi8029 - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7352 24-Core Processor (zen2), Python 2.7.5
See https://gist.github.com/07da567e3b0b01da99daebf3effa6061 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bask-pg0309u04a - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/b4baae5dfffebecb7f9ff987396c9a8e for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @terjekv!

**Pull request merged**